utils/configcheck.py

```diff
@@ -52,6 +56,29 @@ def sd_configcheck(agentConfig):
         configs = load_check_directory(agentConfig, get_hostname(agentConfig))
         get_sd_configcheck(agentConfig, configs)
 
+def agent_container_inspect():
+    print("-- docker_inspect -- \n")
+    dockerutil = DockerUtil()
+    # Self inspection based on cgroups
+
+    proc_path = dockerutil._docker_root + '/proc/self/cgroup'
+    with open(proc_path, 'r') as fp:
+        lines = map(lambda x: x.split(':'), fp.read().splitlines())
+        subsystems = dict(zip(map(lambda x: x[1], lines), map(dockerutil._parse_subsystem, lines)))
+        for k in subsystems.keys():
+            cgroup_regex_list = re.search('(.*/)+(.+?)$', subsystems[k])
+            num_gr = len(cgroup_regex_list.groups())
+            containerID = cgroup_regex_list.group(num_gr) # On all platforms, the container ID is the last part of the path.
+    try:
+        inspect = dockerutil.client.inspect_container(containerID)
```

Comment:
please use `dockerutil.inspect_container` instead. It's just an alias right now but if we add a cache we want to only update this method to use it.